CONTRIBUTING.md

```diff
@@ -0,0 +1,237 @@
+  - [How to Contribute](#how-to-contribute)
+    - [Discuss](#discuss)
+    - [Create an Issue](#create-an-issue)
+    - [Adhere to Coding Style Guide](#adhere-to-coding-style-guide)
+    - [Submit a Pull Request](#submit-a-pull-request)
+    - [Stale issues and PRs](#stale-issues-and-prs)
+    - [Add Documentation](#add-documentation)
+    - [Report on Flaky Tests](#report-on-flaky-tests)
+  - [Project and Milestone Planning](#project-and-milestone-planning)
+    - [Milestones](#milestones)
+      - [Issues](#issues)
+      - [Pull Requests](#pull-requests)
+    - [Projects](#projects)
+    - [Contact Us](#contact-us)
+
+## Project Description
+
+See [README.md](README.md) for a comprehensive project description.
+
+## Code Of Conduct
+
+See the [Eclipse Code Of Conduct](https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php).
+
+## Eclipse Contributor Agreement
+
+Before your contribution can be accepted by the project, you need to create and electronically sign
+a [Eclipse Contributor Agreement (ECA)](http://www.eclipse.org/legal/ecafaq.php):
+
+1. Log in to the [Eclipse foundation website](https://accounts.eclipse.org/user/login/). You will 
+   need to create an account within the Eclipse Foundation if you have not already done so.
+2. Click on "Eclipse ECA", and complete the form.
+
+Be sure to use the same email address in your Eclipse Account that you intend to use when you commit 
+to GitHub.
+
+## How to Contribute
+
+### Discuss
+
+If you want to share an idea to further enhance
+the project or discuss potential use cases, please feel free to create a discussion at the
+[GitHub Discussions page](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions).
+If you feel there is a bug or an issue, contribute to the discussions in
+[existing issues](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues?q=is%3Aissue+is%3Aopen),
+otherwise [create a new issue](#create-an-issue). 
+
+### Create an Issue
+
+If you have identified a bug or want to formulate a working item that you want to concentrate on, 
+feel free to create a new issue at our project's corresponding
+[GitHub Issues page](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/new).
+
+Before doing so, please consider searching for potentially suitable
+[existing issues](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues?q=is%3Aissue+is%3Aopen).
+
+We also use [GitHub's default label set](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels)
+extended by custom ones to classify issues and improve findability.
+
+If an issue appears to cover changes that will have a (huge) impact on the code base and needs to 
+first be discussed, or if you just have a question regarding the usage of the software, please 
+create a [discussion](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions) 
```

Comment:
This link points to EDC repo discussion page. But I think it should point to this repo discussion page. Right ?